modules/@apostrophecms/image/ui/apos/components/AposMediaManager.vue

```diff
@@ -658,6 +667,29 @@ export default {
       if (ref) {
         this.observeLoadRef();
       }
+    },
+    async handleBatchAction({
+      label, action, requestOptions = {}, messages
+    }) {
+      if (action) {
+        try {
+          await apos.http.post(`${this.moduleOptions.action}/${action}`, {
+            body: {
+              ...requestOptions,
+              _ids: this.checked,
+              messages,
+              type: this.checked.length === 1 ? this.moduleLabels.singular
+                : this.moduleLabels.plural
+            }
+          });
+        } catch (error) {
+          apos.notify('apostrophe:errorBatchOperationNoti', {
```

Comment:
Strange contraction, but I guess it's what we already did so 👍 don't break bc

Reply:
yes this is an existing one..